README.CUDA

```diff
@@ -0,0 +1,40 @@
+Support for NVIDIA GPUs via CUDA is currently experimental and many
+features may be missing or incomplete.
+
+# Compiling pocl with CUDA support
+
+Add `-DENABLE_CUDA=ON` to the CMake command-line.
+
+Assumes `libcuda.so` is on library search path. (TODO: Get CMake to sort
+this out).
+
+Currently requires LLVM 3.9.1, and CUDA 7.5/8.0.
+It may work to some extent with LLVM 3.8, although some features may be broken
+due to the TargetAddressSpace pass.
+
+# Running on a CUDA device
+
+Use `POCL_DEVICES=CUDA` to select only CUDA devices.
+
+Requires `POCL_CUDA_TOOLKIT_PATH` environment variable to be set to the root of
+the CUDA installation (containing the `nvvm` directory).
+
+The `POCL_CUDA_GPU_ARCH` environment variable can be set to override
+the target GPU architecture (e.g. `POCL_CUDA_GPU_ARCH=sm_35`), which
+may be necessary in cases where LLVM doesn't yet support the
+architecture (currently an issue for `sm_32` and `sm_53`).
+
+The `POCL_CUDA_NO_PTX_CACHE` environment variable can be set to `1` to
```

Comment:
It wasn't working before, but I've just fixed the issue, so I'll remove this CUDA specific variable now.